addon/modifiers/did-update.js

```diff
@@ -64,9 +64,18 @@ export default setModifierManager(
     createModifier() {
       return { element: null };
     },
-    installModifier(state, element) {
+    installModifier(state, element, args) {
       // save element into state bucket
       state.element = element;
+      if (gte('3.22.0')) {
+        // consume all args
```

Comment:
@cibernox, you have to do this in `updateModifier` as well

Reply:
Done!